### ISSUE TITLE
[BUG] refactor some old controllers

### ISSUE THREAD

**Issue by marcelfolaron** [Leantime/leantime#608]
One year ago I changed the controller layout from a single run() method handling everything (post, get, put) to use 

Get()
Post()
Put()

methods. This made things a lot cleaner within the controller. The controllers should only be used for view handling, routing and variable transfer from and to services. All other logic should stay in the services.

The constructor can be used to share variables, instantiate the template and do authorizations.

We have quite a few controller that still need to be refactored. This will help us getting to a more flexible auth/role system a lot faster.

Basically the run() method is deprecated and should not be used anymore.

**Comment by x2zdigital** [comment]
@marcelfolaron  Ill forward this to my team, we are looking at refactoring permissions to be more flexible.  This looks like it will get in the way.   What is the best way to contact you?

**Comment by marcelfolaron** [comment]
You can send an email to [my first name] @ leantime.io

**Comment by x2zdigital** [comment]
sent a couple days ago, did you receive?
